### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by maxleiko**
Can you check that this is still working ? Because I don't have a board to test (don't know how to plug it also)

### CODE REVIEW

File: src/main/java/org/thingml/comm/rxtx/adaptation/Adaptations2Commands.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    }
+                } else if (p.getPrimitiveType().equals(AdaptationType.StartInstance.name())) {
+                    cmds.add(new TaskResume(((ComponentInstance) p.getRef()).getName()));
+
+                } else if (p.getPrimitiveType().equals(AdaptationType.StopInstance.name())) {
+                    cmds.add(new TaskSuspend(((ComponentInstance) p.getRef()).getName()));
+                }
+            }
+
+            // TODO Note from Max:
```
Comment (maxleiko) [inline src/main/java/org/thingml/comm/rxtx/adaptation/Adaptations2Commands.java:85]: @brice-morin

**Comment by brice-morin** [comment]
@sdalgard can you try Maxime's fork? It is much cleaner and probably a more solid base to re-start with (easier to understand and to evolve)

**Comment by maxleiko** [comment]
@sdalgard note that has I mentionned in the last mail, I've forgot some code in the SerialInterpreter, so there is some refactoring left to do, but not much

**Comment by sdalgard** [comment]
Done

**Pull request merged**